crates/atuin/src/command/client/search/interactive.rs

```diff
@@ -1174,7 +1174,10 @@ pub async fn history(
     any(target_os = "windows", target_os = "macos", target_os = "linux")
 ))]
 fn set_clipboard(s: String) {
-    cli_clipboard::set_contents(s).unwrap();
+    let mut ctx = arboard::Clipboard::new().unwrap();
+    ctx.set_text(s.clone()).unwrap();
+    // Use the clipboard context to make sure it is saved
+    ctx.get_text().unwrap();
```

Comment:
The `get_text()` seems to make sure that the clipboard content is saved, solving #2028. But I think it should be in a short loop to check the contents are equal. Any preference on what loop to do here?